$NetBSD: patch-configure,v 1.8 2020/11/29 22:08:56 dbj Exp $

Remove install program overrides on Darwin
Tweak SEARCH_LIB.
Don't call install_name_tool on symlinks
Avoid -Werror.

--- configure.orig	2020-05-02 14:03:18.000000000 +0000
+++ configure
@@ -19327,15 +19327,6 @@ ac_compiler_gnu=$ac_cv_c_compiler_gnu
 fi
 
 fi
-case "${host}" in
-
-                *-*-darwin* )
-        INSTALL_PROGRAM="cp -fp"
-        INSTALL_DATA="cp -fp"
-        ;;
-    *)
-        ;;
-esac
 
 
 if test "$USE_LINUX" = 1 -o "$USE_GNU" = 1; then
@@ -21778,7 +21769,7 @@ fi
 { $as_echo "$as_me:${as_lineno-$LINENO}: result: $wx_cv_std_libfullpath" >&5
 $as_echo "$wx_cv_std_libfullpath" >&6; }
 
-SEARCH_LIB="`echo "$SEARCH_INCLUDE" | sed s@include@$wx_cv_std_libpath@g` $wx_cv_std_libfullpath"
+SEARCH_LIB="`echo "$SEARCH_INCLUDE" | sed s@include@$wx_cv_std_libpath@g` $wx_cv_std_libfullpath @PREFIX@/lib"
 
 if test "$build" != "$host" -a "$GCC" = yes; then
                     if cross_root=`$CC -print-prog-name=ld 2>/dev/null`; then
@@ -28844,7 +28835,7 @@ for dep in \${libnames} ; do
     changes="\${changes} -change \${4}/\${dep} \${3}/\${dep}"
 done
 for i in \${libnames} ; do
-    ${HOST_PREFIX}install_name_tool \${changes} -id \${3}/\${i} \${1}/\${i}
+    [ -h \${1}/\${i} ] || ${HOST_PREFIX}install_name_tool \${changes} -id \${3}/\${i} \${1}/\${i}
 done
 EOF
             chmod +x change-install-names
@@ -35158,9 +35149,6 @@ if ac_fn_cxx_try_compile "$LINENO"; then
 else
 
                                                                                                                                                                         CFLAGS_OLD="$CFLAGS"
-                        if test "$GCC" = yes ; then
-                            CFLAGS="-Werror $CFLAGS"
-                        fi
 
                         cat confdefs.h - <<_ACEOF >conftest.$ac_ext
 /* end confdefs.h.  */
